### PR TITLE
Users/jagamu/wi changes for k8s

### PR DESCRIPTION
**Pull request by jagadishmurugan**
Enable WI for k8s for AIO

Currently "AksEdgeProductUrl" is set by default "C:\\Users\\Public\\msi\\K3s.msi". This should be replace with path for k8s msi or leave it blank. The latest published k8s.msi will be downloaded.
Note: 1.6.384 k8s.msi will work - it does not need any chnages to config unlike k3s.

### CODE REVIEW

File: tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1 [hunk 1]
```diff
@@ -110,7 +153,7 @@ param(
     }
 
     Write-Host "serviceAccountIssuer = $serviceAccountIssuer"
-    Restart-ApiServer -serviceAccountIssuer $serviceAccountIssuer
+    Restart-ApiServer -serviceAccountIssuer $serviceAccountIssuer -useK8s:$useK8s
 
     & $helmPath repo add azure-workload-identity https://azure.github.io/azure-workload-identity/charts
```
Comment (seanyen) [inline tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1:158]: just checking, the lines from 158-160 are temporary, right?

**Review by seanyen — COMMENTED** [review]
I saw in general the script just `exit -1` when errors occur. Perhaps we should have a task to do `Get-AksEdgeLogs` in the failure case.

File: tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1 (hunk 1 above)
Comment (jagadishmurugan) [reply tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1:158]: yes. wil be removed with final bits from Arc team

**Comment by jagadishmurugan** [comment]
> I saw in general the script just `exit -1` when errors occur. Perhaps we should have a task to do `Get-AksEdgeLogs` in the failure case.
perhaps, created task 28287511

File: tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1 [hunk 2]
```diff
@@ -49,18 +49,58 @@ function Restart-ApiServer
...
+param(
+    [Parameter(Mandatory=$true)]
+    [string] $resourceGroup,
+    [Parameter(Mandatory=$true)]
+    [string] $clusterName,
+    [Parameter(Mandatory=$true)]
+    [string] $subscriptionId
+)
+
+    $retries = 20
```
Comment (seanyen) [inline tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1]: should we try to make the total wait time = 10 mins? According to this doc, the total onboarding could be 5 to 10 mins. https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster?tabs=azure-cli#verify-cluster-connection

Comment (jagadishmurugan) [reply tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1]: lets keep it same as Connect-AksEdgeArc. 600 secs.

**Review by seanyen — APPROVED** [review]
LGTM.

**Pull request merged**